gnomad_qc/v3/sample_qc/subpop_analysis.py

```diff
@@ -53,21 +54,30 @@ def compute_subpop_qc_mt(
         & ~release_ht.was_split
         & hl.is_snp(release_ht.alleles[0], release_ht.alleles[1])
         & hl.is_missing(lcr_intervals.ht()[release_ht.locus])
-    ).key_by("locus")
```

Comment:
Ah, yes, this is needed for `filter_variants`. Since the VDS will be keyed by both. We are using only bi-allelic variants so this will work without a split of the VDS.